src/table/components/virtualized-table/hooks/use-selector.ts

```diff
@@ -0,0 +1,42 @@
+import { Cell } from '../../../../types';
+import { SelectionStates } from '../../../constants';
+import { TableContext, useContextSelector } from '../../../context';
+import { CellStyle } from '../../../types';
+import { getSelectionMouseHandlers } from '../../../utils/handle-click';
+import { getCellSelectionState } from '../../../utils/selections-utils';
+import { getSelectionStyle } from '../../../utils/styling-utils';
+
+const NOOP_ANNOUNCE = () => {};
+
+const PREVENT_DEFAULT = (e: React.MouseEvent<HTMLTableCellElement>) => e.preventDefault();
```

Comment:
is it common practice to do these in snake case?

Reply:
Actually no, will fix